src/repository/storage/postgres/orm/sequelize/noteRelations.ts

```diff
@@ -163,6 +163,24 @@ export default class NoteRelationsSequelizeStorage {
     return affectedRows > 0;
   }
 
+  /**
+   * Delete only parent note relation
+   *
+   * @param noteId - id of the current note
+   */
+  public async deleteParentRelationByNoteId(noteId: NoteInternalId): Promise<boolean> {
+    const affectedRows = await this.model.destroy({
+      where: {
+        noteId: noteId,
+      },
+    });
+
+    /**
+     * If parent relation was not found return false
+     */
+    return affectedRows == 1;
```

Comment:
```suggestion
    return affectedRows > 0;
```

Reply:
But we cant delete more than one row, because note has only one parent